newIDE/app/src/UI/Dialog/index.js

```diff
@@ -39,7 +39,7 @@ type Props = {|
   secondaryActions?: React.Node,
   onRequestClose?: () => void,
 
-  modal?: boolean, // Force the user to use one of the actions in the Dialog. Clicking outside the Dialog will not trigger the onRequestClose.
+  modal?: boolean, //Changes default behaviour of dialog. Clicking outside the dialog box will trigger the onRequestClose.
```

Comment:
> Changes default behaviour of dialog

=> this comment is not providing any information.

Please update the comment to include only necessary information.

Reply:
modified